docs/source/reference/tpu.rst

```diff
@@ -48,7 +50,7 @@ More details can be found on GCP `documentation <https://cloud.google.com/tpu/do
 TPU VMs
 -------
 
-To use TPU VMs, set the following in a task YAML's ``resources`` field: 
+To use TPU VMs, set the following in a task YAML's ``resources`` field:
```

Comment:
nit: in the future, we probably need to add the `--tpu-vm` for `sky launch` as well.

Reply:
Or we can switch the default from tpu nodes to tpu vms @infwinston